eng/common/pipelines/templates/steps/cosmos-emulator.yml

```diff
@@ -9,8 +9,92 @@ steps:
       Write-Host "Target Dir: $targetDir"
       msiexec /a ${{ parameters.EmulatorMsiUrl }} TARGETDIR=$targetDir /qn | wait-process
     displayName: Download and Extract Public Cosmos DB Emulator
+  - powershell: |
```

Comment:
@FabianMeiswinkel Please can you make the changes in this file against the `azure-sdk-tools` repo. Follow instruction for common tooling https://github.com/Azure/azure-sdk-tools/blob/master/doc/common/common_engsys.md.
Your current changes will be overwrite by this sync PR https://github.com/Azure/azure-sdk-for-java/pull/18879/files

Reply:
@chidozieononiwu I'm not sure I understand your request... Why will these files be overwritten by some other content sitting in a different repo? It sounds to me like a very opaque process and out of our own control.
Please do not override these settings unless your other PR preserves them as is.